test/functional/users.js

```diff
@@ -902,6 +902,104 @@ describe("UsersController", function() {
     })
   })
 
+  describe('#updateProfileFrontendPreferences', function() {
+    var authToken
+      , user
+
+    beforeEach(funcTestHelper.createUser('Luna', 'password', function (token, luna) {
+      authToken = token
+      user = luna
+    }))
+
+    it('should store user preferences in db', async () => {
```

Comment:
Another test I'd like to see here: "support for old clients". I.e. updating other user properties (without sending `frontendPreferences`) should not change existing `frontendPreferences`.

See https://github.com/FreeFeed/freefeed-server/blob/development/test/functional/groups.js#L232-L291